src/vstart.sh

```diff
@@ -425,6 +429,12 @@ wconf() {
 	fi
 }
 
+get_nvmeserialnumber() {
```

Comment:
might want to use `get_nvme_serial_number` instead for better readability.

Reply:
Thanks. The name is better. :)